config_test.go

```diff
@@ -37,6 +37,18 @@ func TestReadBadConfig(t *testing.T) {
 	assert.Equal(t, c, Config{}, "Parsing invalid config file should return zero struct")
 }
 
+func TestReadBadConfig_EnvVarsAreNotRead(t *testing.T) {
+	os.Setenv("VENEUR_HOSTNAME", "cux")
+	defer os.Unsetenv("VENEUR_HOSTNAME")
```

Comment:
Hm, I'm mildly worried about setting and unsetting environment variables in tests that could run parallel with other things (even though you don't use `t.Parallel()` here) - couldn't find `envconfig` testing recommendations either, but I guess it's probably fine as long as we never parallelize.

Cc @aditya-stripe for a second opinion

Reply:
Thanks for reviewing this!

Environment variables are in effect global variables, and these IMO should be accessed only from the most exterior layer of the app, and then passed as arguments to more interior layers.  Having a package with "constants" might be helpful if we don't want to "pollute" the signatures of the functions. In Go I know this cannot be enforced at compile time since `const`s are only for values known at compile time, but an alternative would be having functions instead that return the desired environment variables values. It's just an idea :)